wallet/src/de/schildbach/wallet/ui/dashpay/CreateIdentityService.kt

```diff
@@ -585,7 +528,6 @@ class CreateIdentityService : LifecycleService() {
             //
             // Step 3: Verify that the identity was registered
             //
-            //platformRepo.verifyIdentityRegisteredAsync(blockchainIdentity)
```

Comment:
Since we are using the proofs returned by queries and broadcasts, we don't need to call these verify functions.

The verify function calls were commented out for some time.